app/assets/stylesheets/overlays/_activity-feed-overlay.scss

```diff
@@ -17,6 +16,8 @@
 .activity-feed-overlay-timestamp {
   font-size: 14px;
   white-space: nowrap;
+  padding-right: 0 !important;
```

Comment:
I'd second that. If padding isn't being overridden without `!important`, see if there's something in the hierarchy that adds padding.